static/llms.txt

```diff
@@ -0,0 +1,289 @@
+# Stellar Developer Documentation
+Stellar is a layer-1 open-source, decentralized, peer-to-peer blockchain network that provides a framework for developers to create applications, issue assets, write smart contracts, and connect to existing financial rails. Stellar is designed to enable creators, innovators, and developers to build projects on the network that can interoperate with each other.
+
+
+[Build](https://developers.stellar.org/docs/build)
```

Comment:
This list of links risks getting out of date.

We could auto generate the list of links. We have a script that generates routes for another purpose, so someone could copy that and modify it to update the end of this file.

Or, validate the links to check if they are broken links.